pkg/lightning/restore/restore.go

```diff
@@ -1435,6 +1439,7 @@ func (rc *Controller) restoreTables(ctx context.Context) error {
 	// finishSchedulers()
 	// cancelFunc(switchBack)
 	// finishFuncCalled = true
+	taskFinished = true
```

Comment:
What is it used for?

Reply:
If current task is exit before lightning finished (maybe met error or by user terminating), we should clean up the task/table meta tables if all other lightning are finished

Reply:
Oh, I see. `finishSchedulers` will be called before this method ending......  Emmmmmm, it seems to be difficult to understand